Formula/cook.rb

```diff
@@ -0,0 +1,34 @@
+class Cook < Formula
+  desc ""
+  homepage ""
+  url "https://github.com/cooklang/CookCLI.git", tag: "v0.0.8"
+  version "0.0.8"
+  license "MIT"
+
+  depends_on xcode: ["10.0", :build]
+
+  patch :DATA
+
+  def install
+    system "make", "prepare", "build_macos"
+    bin.install ".build/#{ENV['HOMEBREW_PROCESSOR']}-apple-macosx/release/cook"
+  end
+
+  test { system "#{bin}/cook", "version" }
+end
+
+__END__
+diff --git a/Makefile b/Makefile
```

Comment:
This doesn't look right, part of `Makefile` from `cooklang/CookCLI` has been included as a diff.

Reply:
That's correct. I was having trouble building because of the lack of a `--disable-sandbox` flag. So I patch the makefile during the build step to add that flag.

Reply:
Though really, I wonder if a better way might be to just run the `swift` compile task in the homebrew formula, rather than patching the makefile and running a make task...